src/index.js

```diff
@@ -308,6 +309,19 @@ class DAPIClient {
   clearBloomFilter(filter) { return this.makeRequestToRandomDAPINode('clearBloomFilter', { filter }); }
 
   getSpvData(filter) { return this.makeRequestToRandomDAPINode('getSpvData', { filter }); }
```

Comment:
What is that?

Reply:
Let's remove old SPV endpoints in the other PR